cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -229,7 +229,15 @@ func ValidateIPNetFromString(subnet string, minAddrs int64, fldPath *field.Path)
 		allErrs = append(allErrs, field.Invalid(fldPath, subnet, "couldn't parse subnet"))
 		return allErrs
 	}
-	numAddresses := ipallocator.RangeSize(svcSubnet)
+	numAddresses, err := ipallocator.RangeSize(svcSubnet)
```

Comment:
This method (ValidateIPNetFromString()) is used to test both the service CIDR and the cluster's pod CIDR (see lines 250 and 252 below). This means that the cluster's pod CIDR is subject to the same size restriction as the service CIDR. Looks like RangeSize() limits IPv6 CIDRs to have subnet masks that must be longer than that of a /96 subnet. I think we need to loosen this restriction for pod CIDRs (maybe allow cluster pod CIDR of /48 or thereabouts if posible?)

Reply:
I have updated the code to allow a /66 ipv6 cidr.